Zeeguu Reader/Zeeguu Reader/ArticleViewOptionsTableViewController.swift

```diff
@@ -155,17 +168,24 @@ class ArticleViewOptionsTableViewController: UITableViewController, UIPopoverPre
 	
 	func changeFontSize(sender: UIStepper) {
 		let action = ZGJavaScriptAction.ChangeFontSize(Int(sender.value))
-		parent.webview.evaluateJavaScript(action.getJavaScriptExpression()) { (result, error) in
-			print("result: \(result)")
-			print("error: \(error)")
-		}
+		parent.webview.executeJavaScriptAction(action)
```

Comment:
this doesn't really work for me. i think readability is overriding our font setting. could it be?